### DISCUSSION TITLE
Airflow schedule skips current run and next schedule

### DISCUSSION THREAD

**Discussion by bcheerla4509** [apache/airflow#27188] [Q&A]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Airflow Version: 1.10.14

We have below schedule for our DAG, we are intended to run the dag on all days except on Saturday.
So the schedule was 00 12 * * 0-5
however we observed that the DAG scheduled for Friday doesn't executed on Saturday, however it doesn't schedule the dag for Sunday's run too. So DAG is being skipped for Saturday and Sunday, both days.!

Please check.

Thanks
Sagar

### What you think should happen instead

![image](https://user-images.githubusercontent.com/20979585/197241064-feed3d2d-1ccc-4f04-acf7-d9ed4c8fafe3.png)
As shown in the picture, 
9th is Sunday .. there is a schedule with 9th which got actually executed on 10th
10th is Monday .. there is a schedule with 10th which got actually executed on 11th
11th is Tuesday .. there is a schedule with 11th which got actually executed on 12th
12th is Wednesday .. there is a schedule with 12th which got actually executed on 13th
13th is Thursday .. there is a schedule with 13th which got actually executed on 14th
**14th is Friday .. there is a schedule with 14th which didn't execute on 15th (As expected)
15th is Saturday .. there is NO schedule with 15th which didn't executed on 16th (Skipped)**
16th is Sunday ..  there is a schedule with 16th which got executed on 17th

### Operating System

CentOS Linux release 7.9.2009 (Core)

### Deployment

Other

### Deployment details

Deployed via Cloudera distribution CDP7.17.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
We no longer investigate reports related to Airflow 1.10 
Airflow 1.10 is end-of-life for long time.
Your issue may have been already resolved in newer versions.

Converting this report to Q&A discussion
